src/ecs/explorer/ecsServiceNode.ts

```diff
@@ -39,9 +39,14 @@ export class EcsServiceNode extends AWSTreeNodeBase implements AWSResourceNode {
         return await makeChildrenNodes({
             getChildNodes: async () => {
                 const containerNames = await this.ecs.getContainerNames(this.service.taskDefinition!)
-                return containerNames.map(
-                    name => new EcsContainerNode(name, this.name, this.parent.arn, this.ecs, this)
-                )
+                const childNodes = containerNames.map(name => {
+                    return new EcsContainerNode(name, this.name, this.parent.arn, this.ecs, this)
```

Comment:
Nit (that I might be wrong about): can this condense to
```
containerNames.map(name => new EcsContainerNode(name, this.name, this.parent.arn, this.ecs, this) )

Reply:
Yep, since it's only returning that line, we can condense this.